.github/workflows/build-pr.yml

```diff
@@ -131,17 +127,12 @@ jobs:
           path: |
             logs-*.json
 
-  build_test_aks_e2e:
-    name: "Build and run e2e Test on AKS"
-    env:
-      AZURE_CLIENT_ID: 814e6e97-120c-4534-b8a9-f1645bc99500
-      AZURE_TENANT_ID: 72f988bf-86f1-41af-91ab-2d7cd011db47
-      AZURE_SUBSCRIPTION_ID: daae1e1a-63dc-454f-825d-b39289070f79
+  build_test_e2e_full:
+    name: "Build and run e2e on full test matrix"
     runs-on: ubuntu-latest
-    if: contains(github.event.pull_request.labels.*.name, 'safe to test') || github.event_name == 'workflow_dispatch' || github.event_name == 'push'
-    timeout-minutes: 30
+    if: contains(github.event.pull_request.labels.*.name, 'safe to test') || github.event_name == 'workflow_dispatch'
```

Comment:
what is the recommended workflow for making sure your changes are compatible with aks e2e tests?

Reply:
i m not following, could you give an example ? thanks!

Reply:
as discussed, add aks test with "safe to test"